util/testleak/leaktest.go

```diff
@@ -31,38 +31,57 @@ import (
 func interestingGoroutines() (gs []string) {
 	buf := make([]byte, 2<<20)
 	buf = buf[:runtime.Stack(buf, true)]
+	ignoreList := []string{
+		"created by github.com/pingcap/tidb.init",
+		"testing.RunTests",
+		"check.(*resultTracker).start",
+		"check.(*suiteRunner).runFunc",
+		"check.(*suiteRunner).parallelRun",
+		"localstore.(*dbStore).scheduler",
+		"tikv.(*noGCHandler).Start",
+		"ddl.(*ddl).start",
+		"ddl.(*delRange).startEmulator",
+		"domain.NewDomain",
+		"testing.(*T).Run",
+		"domain.(*Domain).LoadPrivilegeLoop",
+		"domain.(*Domain).UpdateTableStatsLoop",
+		"testing.Main(",
+		"runtime.goexit",
+		"created by runtime.gc",
+		"interestingGoroutines",
+		"runtime.MHeap_Scavenger",
+		"created by os/signal.init",
+		// these go routines are async terminated, so they may still alive after test end, thus cause
+		// false positive leak failures
+		"google.golang.org/grpc.(*addrConn).resetTransport",
+		"google.golang.org/grpc.(*ccBalancerWrapper).watcher",
+		"github.com/pingcap/goleveldb/leveldb/util.(*BufferPool).drain",
+		"github.com/pingcap/goleveldb/leveldb.(*DB).compactionError",
+		"github.com/pingcap/goleveldb/leveldb.(*DB).mpoolDrain",
+		"go.etcd.io/etcd/v3/pkg/logutil.(*MergeLogger).outputLoop",
```

Comment:
I have found this and `created by os/signal.init` in other failure tests

They are also in etcd's ignore list
https://github.com/etcd-io/etcd/blob/master/pkg/testutil/leak.go#L121-L132